index.js

```diff
@@ -7,6 +7,13 @@ let config = {
     initLog: false,
     unhandledRejection: true,
     uncaughtException: true,
+    logger(msg) {
```

Comment:
Can we add a test for this? Stub out log and error functions and make sure it's routed to the right place